src/PIL/PdfImagePlugin.py

```diff
@@ -82,7 +77,7 @@ def _save(im, fp, filename, save_all=False):
 
     existing_pdf.start_writing()
     existing_pdf.write_header()
-    existing_pdf.write_comment("created by PIL PDF driver " + __version__)
+    existing_pdf.write_comment("created by Pillow PDF driver " + __version__)
```

Comment:
```suggestion
    existing_pdf.write_comment("created by Pillow {} PDF driver".format(__version__))
```
Is it worth swapping this around, so that it is Pillow 7.0 PDF driver, rather than Pillow PDF driver 7.0?

Reply:
Yep, that reads better, thanks.